src/main/java/frc/robot/subsystems/drive/DriveConstants.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.drive;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants;
+
+public class DriveConstants {
```

Comment:
Why are some things in Constants and other things in DriveConstants?

Reply:
Two different people did it different ways and I'll refrain from naming the person that wasn't me